govcd/edgegateway.go

```diff
@@ -39,6 +40,19 @@ func NewEdgeGateway(cli *Client) *EdgeGateway {
 	}
 }
 
+// Struct which covers NAT rule fields
+type NatRule struct {
```

Comment:
Could we use the vCD API type instead of creating a new one? Suggesting to do it like with LB create functions. This avoids additional level of abstraction and all the cons coming with it.

Reply:
IMHO - no. Cause: GatewayNatRule has Interface field, which is Reference object. GatewayNatRule has Xmls field. Description, ruletype, enabled is in NatRule which is parent. Parent Natule has 5 fields which we don't need